src/middlewared/middlewared/plugins/pool.py

```diff
@@ -685,6 +697,10 @@ def add(attr):
         ('edit', _add_inherit('readonly')),
         ('edit', _add_inherit('recordsize')),
         ('edit', _add_inherit('snapdir')),
+        ('add', Inheritable('quota_warning', value=Float('quota_warning', validators=[Range(0.0, 1.0)]))),
+        ('add', Inheritable('quota_critical', value=Float('quota_critical', validators=[Range(0.0, 1.0)]))),
+        ('add', Inheritable('refquota_warning', value=Float('refquota_warning', validators=[Range(0.0, 1.0)]))),
+        ('add', Inheritable('refquota_critical', value=Float('refquota_critical', validators=[Range(0.0, 1.0)]))),
```

Comment:
Why Inheritable?

I dont see it being used anywhere else so maybe does not make sense to make it a global schema?
Perhaps it would be easier/simpler to treat "null" as INHERIT?

Reply:
> Perhaps it would be easier/simpler to treat "null" as INHERIT?

It would be easier, but it also would be inconsistent with other properties.